test/Npgsql.Tests/SchemaTests.cs

```diff
@@ -7,10 +7,9 @@
 using NpgsqlTypes;
 using NUnit.Framework;
 
-#pragma warning disable 8602 // Warning should be removable after rc2 (https://github.com/dotnet/runtime/pull/42215)
-
 namespace Npgsql.Tests
 {
+    [NonParallelizable]
```

Comment:
Most of this should parallelizable right? Just leaving for later?

Reply:
Yes, exactly.